.github/workflows/nf-core-linting.yml

```diff
@@ -0,0 +1,71 @@
+name: nf-core linting
+# This workflow is triggered on pushes and PRs to the repository.
+# It runs the `nf-core lint` tests to ensure that the module code meets the nf-core guidelines
+on: [push, pull_request]
+
+jobs:
+  changes:
+    name: Check for changes
+    runs-on: ubuntu-latest
+    outputs:
+      # Expose matched filters as job 'modules' output variable
+      modules: ${{ steps.filter.outputs.changes }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: 'tests/config/pytest_software.yml'
+
+  test:
+    runs-on: ubuntu-20.04
+    name: ${{ matrix.tags }} ${{ matrix.nxf_version }}
+    needs: changes
+    if: needs.changes.outputs.modules != '[]'
+    strategy:
+      fail-fast: false
+      matrix:
+        nxf_version: ['21.03.0-edge']
```

Comment:
If only planning to use one Nextflow version forever (I think this is fine?) then doesn't need to be in the matrix, can just be in the top-level `env`

Reply:
You read my mind! Was about to post about that on Slack too!

Reply:
Ok. Fixed in  https://github.com/nf-core/modules/pull/414/commits/7902660ff663ca86b5b419dca198abed57e373eb